packages/client3/components/nav/wallet-status.tsx

```diff
@@ -95,10 +94,11 @@ export function WalletStatus({ onWalletDisconnect }: WalletInfoProps) {
   );
 
   return (
-    <div className="w-80 divide-y divide-sand-100">
+    <div className="divide-sand-100 w-80 divide-y">
       {error ? (
-        <HelperText isError>Error while fetching wallet status</HelperText>
-      ) : null}
+        <></>
+      ) : // <HelperText isError>Error while fetching wallet status</HelperText>
+      null}
```

Comment:
Old component used by goldfinch. I commented instead of deleting it for future reference.